aniwash/src/main/java/aniwash/viewmodels/MainViewModel.java

```diff
@@ -123,13 +132,22 @@ public Appointment createAppointment(ZonedDateTime zdtStart, ZonedDateTime zdtEn
         }
         appointment.setTotalPrice(totalPrice);
         appointment.getLocalizations().put("en", localAppointment);
+
+        if( descriptionText.getText() != null && !descriptionText.getText().isEmpty() ) {
+            appointment.getLocalizations().get("en").setDescription(descriptionText.getText());
```

Comment:
You can add this description text in LocalizedAppointment(Appointment, Description)